### DISCUSSION TITLE
Access to dummy variables in internal mice procedure ('pad' element of mids object is missing)

### DISCUSSION THREAD

**Discussion by alexpate30** [amices/mice#626] [Q&A] [answered]
Hello all,

mice allows specification of passive imputation for interaction terms. Interaction terms involving a categorical variable will always need to work with dummy variables. It is recommend accessing the dummy versions of the categorical variables that are used internally in mice, see section 3.4: passive imputation, top of page 30 in previously linked manuscript 
 (https://www.jstatsoft.org/article/view/v045i03).

"Interactions involving categorical variables need a representation using dummy variables. The mice() function internally creates dummy variables for any factor that are being used as a predictor. The data and names of these dummy variables can be accessed from imp$pad$data."

I cannot seem to access the pad element of the mice object. Has this been removed? Has it been transferred into another element of the mice object? I note this was fixed as a bug in mice 2.4.1: https://github.com/amices/mice/blob/master/NEWS.md

Many thanks for any assistance!

Reproducible example taken directly from linked manuscript. R version 4.3.1, using mice version 3.16.0 (as available on CRAN):

```
install.packages("mice")
library(mice)

str(nhanes2)
nhanes2.ext <- cbind(nhanes2, bmi.chl = NA)
ini <- mice(nhanes2.ext, max = 0, print = FALSE)
head(ini$pad$data, 3)
ini$pad
str(ini)
```

**Comment by hanneoberman** [comment, marked as answer]
Hi @alexpate30, you're right, this functionality is superseded. See https://github.com/amices/mice/issues/235#issuecomment-632997015

**Reply by alexpate30** [reply]
Many thanks for pointing me in the direction for this, much appreciated.
